olympus-oir-pybind11/read_oir_pybind11.py

```diff
@@ -97,13 +97,10 @@ def main(filepath):
     # Read Image Stack
     # ==============================
 
-    # TODO: Requires implementation.
-
-    pAxes = (IDA_AXIS_INFO * 3)()
+    pAxes = ida_sample_lib.VectorAxisInfo(3)
     nAxisCount = 1  # dummy
 
     # Create Frame Manager
-    # TODO: Arguments must be of the appropriate type (pAxes)
     frame_manager = ida_sample_lib.FrameManager(
```

Comment:
@tuanna26 
Can you also port the tiff image output code to confirm that frame_manager.GetImageBody(rect) is working properly?

Reply:
@itutu-tienday 
I saw that there had tiff image converted in `main/olympus-oir/read_oir.py::144`. Do you mean this source code will be picked into this file?
Or which tiff image output source code do you want to be converted here?

Reply:
@tuanna26 
>  Do you mean this source code will be picked into this file?

Yes, basically, if you could port the above code as is, I think it would be OK.
*If you have any problems, please correct or contact us.

Reply:
@itutu-tienday 
I have done.
Please check it